### ISSUE TITLE
k3s scan reports failure when not using etcd backend

### ISSUE THREAD

**Issue by rancher-max** [rancher/cis-operator#100]
Running a k3s scan (both hardened and permissive) reports as a failure when k3s cluster is using a backend other than etcd. Screenshot below showing hardened scan results:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55997940/124651117-639dca00-de4f-11eb-8ffb-56b95a01c216.png">

These checks are all related to etcd and should not be run when the backend is not etcd, as they would become Not Applicable for a k3s cluster at that point.

**Comment by deniseschannon** [comment]
Available in the CIS scan chart v2.0.1 with `rancher/security-scan:v0.2.4-rc1`

**Comment by sowmyav27** [comment]
**On 2.6-head commit id: `fe57abd`**

- Deployed CIS 2.0.1
```
image:
  cisoperator:
    repository: rancher/cis-operator
    tag: v1.0.6-rc1
  securityScan:
    repository: rancher/security-scan
    tag: v0.2.4-rc2
  sonobuoy:
    repository: rancher/mirrored-sonobuoy-sonobuoy
    tag: v0.53.2
```

- Deployed an imported k3s HA hardened cluster with `externaldb` - 2 server nodes, 3 worker nodes
- `k3s-cis-1.6-profile-hardened` scan on this cluster passed. No failures seen

**Comment by rancher-max** [comment]
I'm seeing weird errors here on both etcd and "other" clusters now. In both hardened and permissive runs on an etcd cluster, I see the same 7 failures mentioned above:

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/55997940/134990311-f869274b-0cf8-428d-882c-f2755a2435a7.png">


Also, sporadically, I see errors running sonobuoy:
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/55997940/135006612-22321eb6-1c44-4d2a-8e84-f5fc20f1ace5.png">

Confirmed this is happening on both 2.6-head commit `06e19b0` and v2.6.1-rc3, using the same k3s and CIS versions mentioned in Sowmya's [comment](https://github.com/rancher/cis-operator/issues/100#issuecomment-928379489) above

**Comment by dereknola** [comment]
This has been narrowed down to an issue with only external databases and HA etcd. So the previous PRs did not completely fix the issue. A further PR will be needed.

**Comment by rancher-max** [comment]
It appears this has been resolved in CIS App v2.0.2 using rancher/security-scan:v0.2.5-rc1. I no longer see these mixed errors, and the "error running sonobuoy" is being tracked by a different issue.
